### DISCUSSION TITLE
Is there a way to interact with Ldap server to handle CRUD actions like Spring Data LDAP

### DISCUSSION THREAD

**Discussion by josephassiga** [quarkusio/quarkus#23885] [Community]
I want to use Quarkus to develop my API, i have one API that will interact with Ldap, but i don't find any library that can do the same as Spring Ldap or Spring Data Ldap.

Please could you suggest me how i could do it with Quarkus ?

Regards

**Comment by maxandersen** [comment]
I don't know of any Quarkus optimized ones but dependent on your usecase using something like apache ldap libraries or UnboundID LDAP SDK would be options. those are some of the libraries spring ldap uses behind the scenes.

Those should work as is in Quarkus jvm mode.
